doc/examples/reconst_csa.py

```diff
@@ -41,8 +41,8 @@
 from dipy.segment.mask import median_otsu
 
 
-maskdata, mask = median_otsu(data, vol_idx=range(10, 50), median_radius=3, numpass=1,
-                             autocrop=True, dilate=2)
+maskdata, mask = median_otsu(data, vol_idx=range(10, 50), median_radius=3,
```

Comment:
This is weird. I believe that the `vol_idx` here  (and elsewhere using this data) should be just the values where the `gtab.b0s_index` is `True`. Or am I missing  something?

Reply:
Yes, technically it should be where `gtab.b0s_index` is `True` if you want to perform skull stripping based upon the b0s. The skull is still present in the DWIs so I guess it still works with the `range(10, 50)`.